### ISSUE TITLE
Mobile number slider no keyboard

### ISSUE THREAD

**Issue by jessb321** [EDCD/coriolis#248]
On iOS the number sliders don't pop up a keyboard.

**Comment by pnellesen** [comment]
I can take a look at this when I get some time - I notice that (on my iPhone, at least) there's no way to interact with the slider if you're in Voiceover. The cursor will move to the various labels but there doesn't appear to be a way to change them the value itself.  (I do have a fair bit of experience debugging these kinds of issues at work for other controls, but not all that familiar with sliders)

**Comment by willyb321** [comment]
Yeah, was talking with @urukcan this morning, he mentioned that on iOS the number sliders don't pop up a keyboard.

**Comment by pnellesen** [comment]
Started looking at this - I can get keyboard focus (desktop) to go to the sliders simply by adding "tabIndex="0" to the <svg> element in Slider.jsx. Nothing happens when a key is pressed, but it's a start, at least ;) May need to add a keydown handler similar to what they did here: https://github.com/react-component/slider/issues/131#issuecomment-308567343

**Comment by pnellesen** [comment]
Got this working reasonably well - input field is 0x0, numeric-only, and is positioned middle-left of the slider element. Keyboard will appear on touchStart event. The page scrolls up, but you can still manipulate the slider without removing finger from screen. Will add some logic to set min/max values and update value as numbers are typed.

**Comment by pnellesen** [comment]
Tap/Hold function is working well on Android, but my iPhone Safari will not set focus to the text box when using a setTimeout to do it (apparently an old issue: https://stackoverflow.com/questions/6866433/focus-not-working-within-settimeout-on-ios).  Focus works fine if setting immediately on touchStart.
